### ISSUE TITLE
Define Show interface on Integers

### ISSUE THREAD

**Issue by oxinabox** [google-research/dex-lang#311]
Consider:
```
:p show 10.0
(AsList 7 ['1', '0', '.', '0', '0', '0', '0'])

:p show 10
Type error:Couldn't synthesize a class dictionary for: (Show Int32)

:p show 10
   ^^^^^
```

It would be useful to be able to convert integers to strings.

**Comment by dan-zheng** [comment]
I'm happy to look into this.

**Comment by apaszke** [comment]
Implemented in #318!
